staging/src/k8s.io/client-go/rest/transport.go

```diff
@@ -89,12 +91,22 @@ func (c *Config) TransportConfig() (*transport.Config, error) {
 		Proxy: c.Proxy,
 	}
 
-	if c.ExecProvider != nil && c.AuthProvider != nil {
+	if c.Exec.ExecProvider != nil && c.AuthProvider != nil {
 		return nil, errors.New("execProvider and authProvider cannot be used in combination")
 	}
 
-	if c.ExecProvider != nil {
-		provider, err := exec.GetAuthenticator(c.ExecProvider)
+	if c.Exec.ExecProvider != nil {
+		caData, err := dataFromSliceOrFile(c.CAData, c.CAFile)
```

Comment:
only load this and fail if the provider requested cluster data, right?

Reply:
https://github.com/kubernetes/kubernetes/pull/95489/commits/bd4af0df52b7da2a838795ccf9a8760a0316da48